api/platformpb/platform.proto

```diff
@@ -158,4 +171,15 @@ service Platform {
       description: "SearchOrganizationEntitlements fetches details of the entitlement's available to the Portal organization that the PMM server is connected to."
     };
   }
+  // GetContactInformation fetches the contact details of the customer success employee handling the Percona customer account from Percona Platform.
+  rpc GetContactInformation(GetContactInformationRequest) returns (GetContactInformationResponse) {
+    option (google.api.http) = {
+      post: "/v1/Platform/GetContactInformation"
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      summary: "GetContactInformation"
```

Comment:
Little improvement following the goals from https://jira.percona.com/browse/PMM-9378 and https://github.com/percona/pmm/pull/823 

```suggestion
      summary: "Get Contact Information"
```